librz/asm/arch/rl78/rl78.c

```diff
@@ -0,0 +1,142 @@
+// SPDX-FileCopyrightText: 2023 Bastian Engel <bastian.engel00@gmail.com>
+// SPDX-License-Identifier: LGPL-3.0-only
+
+#include "rl78.h"
+
+#include "maps.h"
+
+static bool parse_operand(struct rl78_operand *operand, size_t *p,
+                          const ut8 *buf, size_t buf_len);
+
+static bool optype_es_applies(enum rl78_operand_type type);
+
+bool rl78_dis(struct rl78_instr RZ_OUT *instr, size_t RZ_OUT *bytes_read,
+              const ut8 *buf, size_t buf_len)
+{
+        if (buf_len == 0) {
+                *bytes_read = 0;
+                return false;
+        }
+
+        // points to the next byte to be processed
+        size_t p = 0;
```

Comment:
Rename `p` -> `next_byte_idx` or something other meaningful please. Then the comment is also not necessary.